docs/central-entities.rst

```diff
@@ -159,21 +166,19 @@ Test your Form to make sure it works and collects the data that you need.
 
 .. _central-entities-registration-forms-destination:
 
-Specify the Dataset the Form should save Entities to
-""""""""""""""""""""""""""""""""""""""""""""""""""""
+Specify the Entity List the Form should save Entities to
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 
-Add a new ``entities`` sheet to your XLSForm. This is where you will specify your Dataset’s name, under ``dataset``.
+Add a new ``entities`` sheet to your XLSForm. This is where you will specify your Entity List name, under ``list_name``.
```

Comment:
@ktuite I'm specifically interested in your thoughts on `list_name` since you've made educational materials for all this. Do you think it's helpful or harmful that it's the same as the column header to specify the list that a choice is in from the `choices` sheet?

I thought it was helpful but I'm questioning that a little bit because the usage is different. On the `choices` tab, we're explicitly stating the list that a concrete item belongs in. Here, we're generally declaring the list that abstract entities will be created in. Maybe that's similar enough?

Reply:
i think `list_name` is okay. the entities tab has such a different purpose and shape than the choices tab that i think it's ok for the column header to be the same but have slightly different meanings within the different context. this happens with `label`, too. having some other word that is slightly different yet also isn't a perfect fit seems like it would be more confusing